app/api/groups.py

```diff
@@ -28,9 +29,16 @@ def before_post(self, args, kwargs, data):
         :return:
         """
         data['user'] = current_user.id
+        require_relationship(['event'], data)
```

Comment:
Not required. 

1. You should be able to create group without event 
2. Group doesn't have one event. It has events, if the schema field is named event, please change it to events